src/applications/ivc-champva/10-10D/tests/unit/config/submitTransformer.unit.spec.js

```diff
@@ -72,14 +46,15 @@ describe('transform for submit', () => {
   });
   it('should attach applicant name to each uploaded file', () => {
     const modified = JSON.parse(JSON.stringify(mockData));
-    modified.data.applicants[0].applicantMedicareCardFront = [
+    const fileKey = Object.keys(REQUIRED_FILES)[0]; // grab a file we expect to be uploaded
+    modified.data.applicants[0][fileKey] = [
```

Comment:
Updated this test so it always uses a legitimate file keyname from the `REQUIRED_FILES` list. This was necessary because the `submitTransformer` logic ignores any file properties that are not listed in that constants object.